modules/python/src2/cv2_convert.hpp

```diff
@@ -268,6 +268,11 @@ PyObject* pyopencv_from(const std::vector<Tp>& value)
 template<typename K, typename V>
 bool pyopencv_to(PyObject *obj, std::map<K,V> &map, const ArgInfo& info)
 {
+    if (!obj || obj == Py_None)
+    {
+        return true;
+    }
```

Comment:
@VadimLevin Is is relevant patch?

Reply:
I guess it's the common check that I missed. It's used for all other wrappers.

The code below won't work without this check.
```
GAPI_EXPORTS_W void foo(const map<int, int> &map = {});
```